scripts/stablex/utilities.js

```diff
@@ -53,9 +86,64 @@ const getOrdersPaginated = async (instance, pageSize) => {
   return orders
 }
 
+const sendLiquidityOrders = async function(
+  instance,
+  tokenIds,
+  PRICE_FOR_LIQUIDITY_PROVISION,
+  SELL_ORDER_AMOUNT_OWL,
+  artifacts,
+  OWL_NUMBER_DIGITS = 18
+) {
+  const maxUint32 = new BN(2).pow(new BN(32)).sub(new BN(1))
+
+  const minBuy = []
+
+  for (const tokenId of tokenIds) {
+    const numberOfDigits = (await fetchTokenInfo(instance, [tokenId], artifacts))[tokenId].decimals
```

Comment:
I would propose fetching all the token info before the loop, since this helper method returns an object indexed by ID... I suppose it doesn't make a difference.